server/models/Profile.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
+const { Availability } = require("./Availability");
+const { User } = require("./User");
+
+const profileSchema = new mongoose.Schema({
+  firstName: {
+    type: String,
+    required: true,
+  },
+  lastName: {
+    type: String,
+    required: true,
+  },
+  gender: {
+    type: String,
+    required: true,
+  },
+  dateOfBirth: {
+    type: Date,
+  },
+  profilePicture: {
+    type: String,
+  },
+  phoneNumber: {
+    type: Number,
+  },
+  address: {
+    type: String,
+  },
+  description: {
+    type: String,
+  },
+  petSitter: {
+    type: Boolean,
+  },
+  rate: {
+    type: Number,
+    required: true,
```

Comment:
I don't think we want to have this as a required field (for example, an owner does not need to specify a price)